### PR TITLE
LPS-125132 buildLang

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://issues.liferay.com/browse/LPS-125132

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by natocesarrego** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtil.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+				(++count == maximumSubmissionsForGuestUploadFields)) {
+
+				return true;
+			}
+		}
+
+		return false;
+	}
+
+	protected static boolean hasGuestUploadField(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtil.java:87]: Should this method and the following be `protected` or can we make it `private`?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+		_ddmForm.addDDMFormField(
+			new DDMFormField("fieldName", "document_library"));
+
+		Assert.assertFalse(
+			_ddmFormGuestUploadFieldUtil.hasGuestUploadField(
+				mockDDMFormInstance()));
+	}
+
+	@Test
+	public void testIsMaxSubmissionLimitReachedWithGuestUserAndUploadFieldNotAllowedForGuests()
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtilTest.java:91]: @carolmariaabb, from previous experiences, I believe we'll need to have shorter tests' names.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+		throws Exception {
+
+		_ddmForm.addDDMFormField(
+			new DDMFormField("fieldName", "document_library"));
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormGuestUploadFieldUtilTest.java:95]: @carolmariaabb, can we reuse the method `addGuestUploadField` here? If we add a parameter to this method, I understand that we can reuse it. What do you think?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadResponseHandler.java [hunk 4]
```diff
@@ -75,14 +74,12 @@ else if (portalException instanceof FileSizeException) {
 		else if (portalException instanceof InvalidFileException) {
 			errorMessage = themeDisplay.translate("please-enter-a-valid-file");
 		}
+		else {
+			errorMessage = themeDisplay.translate(
+				"an-unexpected-error-occurred-while-uploading-your-file");
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadResponseHandler.java:79]: @carolmariaabb, what kind of generic error can occur?

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
Resent [here](https://github.com/natocesarrego/liferay-portal/pull/947).

**Comment by carolmariaabb** [comment]
ci:stop

**Pull request closed without merging**